docs/jupyter-lite.json

```diff
@@ -1,14 +1,5 @@
 {
-    "LiteBuildConfig": {
```

Comment:
This `LiteBuildConfig` config might have been placed in this file by mistake.

Normally it should go to `jupyterlite_config.json`. But for the docs environment it's probably not needed for now.